app/artifact-cas/internal/server/grpc.go

```diff
@@ -93,6 +94,18 @@ func NewGRPCServer(c *conf.Server, authConf *conf.Auth, byteService *service.Byt
 	if c.Grpc.Timeout != nil {
 		opts = append(opts, grpc.Timeout(c.Grpc.Timeout.AsDuration()))
 	}
+	cert := c.Grpc.TlsConfig.Certificate
```

Comment:
it might make sense to use the `secure getter` `getTLSConfig.GetCertificate` this will prevent panics if the TLSConfig is not present, which will not in currently deployed instances.

Reply:
or something like `if tlsConf := c.Grpc.GetTLSConfig(); tlsConf != nil` { /load and append of the key / }

Reply:
Make sense 👍